diff --git a/masonry.install b/masonry.install
index 2301368..c78a56d 100644
--- a/masonry.install
+++ b/masonry.install
@@ -5,6 +5,7 @@
  * 
  * Sponsored by: www.freelance-drupal.com
  */
+use Drupal\Core\Cache\Cache;
 
 /**
  * Implements hook_requirements().
@@ -50,3 +51,9 @@ function masonry_requirements($phase) {
   return $requirements;
 }
 
+/**
+ * Clean cache for reloading missing library dependency.
+ */
+function masonry_update_8001(&$sandbox) {
+  Cache::invalidateTags(['rendered']);
+}
diff --git a/masonry.libraries.yml b/masonry.libraries.yml
index 0c8c730..2302f28 100644
--- a/masonry.libraries.yml
+++ b/masonry.libraries.yml
@@ -9,4 +9,5 @@ masonry.layout:
     masonry: []
   dependencies:
     - core/jquery
-    - core/drupal
\ No newline at end of file
+    - core/jquery.once
+    - core/drupal
